docs/book/src/component-config-tutorial/generate_docs.go

```diff
@@ -0,0 +1,196 @@
+/*
```

Comment:
Could we move this to docs/book/src/hack/internal/generate_component_config.go?
Because then:
- we have the script under the hack, which makes it clear that is a script/helper
- we do not allow anybody to import this code implementation.

Reply:
Make sense 👍